lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -660,6 +660,38 @@ def add_attr(item):
     cube.attributes["institution"] = "Met Office"
 
 
+def table_1_attributes(cube, field):
+    """Add attributes to the cube."""
+
+    def add_attr(item):
+        """Add an attribute to the cube."""
+        if hasattr(field, item):
+            value = getattr(field, item)
+            if is_missing(field, value):
+                return
+            if "radius" in item:
+                value = f"{value} km"
```

Comment:
Unlike the `attributes()` method, none of the attributes added here include `radius` in their keys.